⏣ Version 1.1

tooxo opened the pull request:
Removes "lyrics", "official video" etc.

Commit:
Added Title Strip

pep8speaks commented:
Hello @tooxo! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`discord/discord_music.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py):

> [Line 60:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L60): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 156:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L156): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 157:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L157): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 163:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L163): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 164:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L164): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 168:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L168): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 170:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L170): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 171:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L171): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 177:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L177): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 179:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L179): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 180:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L180): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 633:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L633): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 635:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L635): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 637:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L637): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 639:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L639): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 641:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L641): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 644:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L644): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 795:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L795): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 916:29](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L916): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 925:116](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/discord_music.py#L925): [E203](https://duckduckgo.com/?q=pep8%20E203) whitespace before ':'









* In the file [`discord/test.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/test.py):

> [Line 66:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/edbfff01e1ceb6330895bd557cacce0e8982633e/discord/test.py#L66): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator









##### Comment last updated at 2019-09-01 14:04:44 UTC

Commit:
Added Better Sani

Commit:
Added better url sanitization
Fixed bugs

Commit:
Add TravisBuddy

tooxo committed:
Merge branch 'master' into add_title_strip

Commit:
⏣ Version 1.1

- Added Detection of empty Channel
- Added "after-playing-done" Message
- Added Error Message for multiple Errors
- Rewritten Objects, clearing up code structure
- Code Cleanup
- Rewritten Help Command
- Cover Search by Spotify Enabled
- Fixed Spelling Errors
- Fixed Bug with Spaces in Search Term
- Extended README

Commit:
Merge remote-tracking branch 'origin/add_title_strip' into add_title_strip

tooxo commented:
@pep8speaks suggest diff

pep8speaks commented:
Here you go with [the gist](https://gist.github.com/bc990ecb5e808423607da8164fe6f24b) !

> You can ask me to create a PR against this branch with those fixes. Simply comment `@pep8speaks pep8ify`.

 @tooxo

tooxo commented:
@pep8speaks pep8ify
